src/Pages/GlobalConfigurations/Authorization/README.md

```diff
@@ -0,0 +1,40 @@
+```mermaid
+graph TD;
+  Authorization-->SSOLoginServices;
+  Authorization-->UserPermissions;
+  Authorization-->PermissionGroups;
+  Authorization-->APITokens;
+
+  subgraph "Authorization Provider"
+    UserPermissions-->UserPermissionsList;
+    UserPermissions-->UserPermissionsAddEdit;
+
+    UserPermissionsList-->UserPermissionListHeader;
+    UserPermissionsList-->UserPermissionRow;
+
+    UserPermissionsAddEdit-->UserForm;
+    UserForm-->AppPermissions;
```

Comment:
GroupsPermissions should also be added here

Reply:
Not adding for now since the GroupsPermissions is only being used by API Tokens as now.